packages/expo-background-task/src/BackgroundTask.ts

```diff
@@ -0,0 +1,104 @@
+import { UnavailabilityError } from 'expo-modules-core';
+import * as TaskManager from 'expo-task-manager';
+
+import { BackgroundTaskOptions, BackgroundTaskStatus } from './BackgroundTask.types';
+import ExpoBackgroundTaskModule from './ExpoBackgroundTaskModule';
+
+// @needsAudit
+/**
+ * Returns the status for the Background Task API. On web, it always returns `BackgroundTaskStatus.Restricted`,
+ * while on native platforms it returns `BackgroundTaskStatus.Available`. There is
```

Comment:
```suggestion
 * while on native platforms it returns `BackgroundTaskStatus.Available`.
```

Partially unfinished sentence.